package.json

```diff
@@ -29,18 +29,18 @@
     "test": "jest"
   },
   "dependencies": {
-    "node-fetch": "^2.6.0",
-    "@types/node-fetch": "^2.5.7",
     "@types/node": "*",
-    "form-data": "^2.5.0",
-    "json5": "^2.2.3",
+    "@types/node-fetch": "^2.5.7",
     "btoa": "^1.2.1",
+    "crypto": "^1.0.1",
     "es6-promise": "^4.2.4",
-    "url-parse": "^1.4.3",
-    "reflect-metadata": "^0.1.13",
-    "qs": "^6.11.1",
+    "form-data": "^2.5.0",
+    "json5": "^2.2.3",
     "moment": "^2.29.4",
-    "crypto": "^1.0.1"
+    "node-fetch": "^2.6.0",
+    "qs": "^6.11.1",
+    "reflect-metadata": "^0.1.13",
+    "url-parse": "^1.4.3"
   },
```

Comment:
can we update the license field in package.json to be MIT? https://docs.npmjs.com/cli/v9/configuring-npm/package-json#license

Reply:
Hey @katebrune - This has been addressed in the latest push